client/components/Byline/Byline.tsx

```diff
@@ -3,12 +3,20 @@ import React from 'react';
 import ensureUserForAttribution from 'utils/ensureUserForAttribution';
 import { joinOxford, naivePluralize } from 'utils/strings';
 import { AttributableUser, User } from 'types';
+import { Tooltip, Icon } from '@blueprintjs/core';
 
 const isFullUser = (u: AttributableUser): u is User => 'slug' in u;
 
+export enum BylineOption {
+	By = 'by',
+	Contributors = 'Contributors (No Particular Order)',
+	Az = 'Contibutors (A-Z)',
+	Za = 'Contributors (Z-A)',
+}
+
 export type BylineProps = {
 	ampersand?: boolean;
-	bylinePrefix?: null | string;
+	bylinePrefix?: null | string | BylineOption;
```

Comment:
This is expressing something a bit muddled — the `bylinePrefix` is supposed to be "the thing before the byline" but it's now "the way we sort byline contributors". Even though these things are related, I think they need to remain as two separable options — `bylinePrefix` and `bylineOrdering` — because there are places in the codebase such as Pub Connections cards where we set the `bylinePrefix` to `null`, and, best I can tell, this would override that.